### PR TITLE
Temporary fix for License notifications

### PR DESCRIPTION
**Pull request by bamandel**
License notifications do not have groups associated with them. Because of this, we need to check if the message is about licenses then add all the emails associated with the job.

This is meant to be temporary until we rework the way providers handle the Email channel.

### CODE REVIEW

File: src/main/java/com/synopsys/integration/alert/provider/DefaultEmailHandler.java [hunk 1]
```diff
@@ -93,4 +101,32 @@ public FieldAccessor updateFieldAccessor(final AggregateMessageContent content,
         }
         return Set.of();
     }
+
+    // FIXME temporary fix for license notifications before we rewrite the way emails are handled in our workflow
+    private Set<String> licenseNotificationCheck(final AggregateMessageContent content, final FieldAccessor fieldAccessor, final boolean projectOwnerOnly) {
+        final Optional<LinkableItem> subTopic = content.getSubTopic();
+        if (subTopic.isEmpty()) {
+            final Boolean filterByProject = fieldAccessor.getBoolean(CommonDistributionConfiguration.KEY_FILTER_BY_PROJECT).orElse(false);
```
Comment (gkillough) [inline src/main/java/com/synopsys/integration/alert/provider/DefaultEmailHandler.java]: This isn't for this pull request, but in some major version, we should have filterByProject default to true for email. Otherwise, a mistake could generate lots of unwanted emails.

File: src/main/js/component/audit/Index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { BootstrapTable, ButtonGroup, TableHeaderColumn } from 'react-bootstrap-table';
-import { getAuditData } from 'store/actions/audit';
+import { getAuditData, resendNotification } from 'store/actions/audit';
```
Comment (psantos1113) [inline src/main/js/component/audit/Index.js]: Why was this added?  Was it missing from Index.js beforehand and causing an error in the javascript console?

Comment (jamesrichard91) [reply src/main/js/component/audit/Index.js]: Yes it was missing before

Comment (bamandel) [reply src/main/js/component/audit/Index.js]: This was because of the issue with My and Gavin's save actions. So I added it as I noticed it was broken when trying to test this feature.
